R/R/prophet.R

```diff
@@ -1305,6 +1444,7 @@ predict.prophet <- function(object, df = NULL, ...) {
   return(df)
 }
 
+
```

Comment:
we should be able to remove the `piecewise_linear` and the `piecewise_logistic` functions with this change, as well as `predict_trend`.